superset/charts/api.py

```diff
@@ -894,3 +900,19 @@ def import_(self) -> Response:
         except Exception as exc:  # pylint: disable=broad-except
             logger.exception("Import chart failed")
             return self.response_500(message=str(exc))
+
+    @expose("/time_range/", methods=["GET"])
+    @safe
+    @rison(get_time_range_schema)
```

Comment:
it would be great to add an OpenAPI spec here, feel free to ping me if you need any help with it

Reply:
Got it. I will add the OpenAPI spec as soon as possible and frontend UT